src/integration-test/java/com/mongodb/jdbc/integration/testharness/DataLoader.java

```diff
@@ -99,6 +99,21 @@ public void dropCollections() {
         }
     }
 
+    public void loadTestRow(String db, String collection, Map<String, Object> row) {
```

Comment:
I don't see this being used anywhere anymore -- is it?

Reply:
Removed the method